src/line_range.rs

```diff
@@ -47,7 +47,16 @@ impl LineRange {
             }
             2 => {
                 new_range.lower = line_numbers[0].parse()?;
-                new_range.upper = line_numbers[1].parse()?;
+
+                new_range.upper = if line_numbers[1].bytes().next().unwrap() == b'+' {
+                    let more_lines = &line_numbers[1][1..]
+                        .parse()
+                        .map_err(|_| "Invalid line number after +")?;
```

Comment:
Maybe "invalid number after"?

Reply:
After a bit more thinking I changed it to "Invalid character after +".
Please let me know if you still think we should change it, I don't mind.
I kept capital `I` because other messages start with capital letter.
And using `character` instead of `number` in the message because there might be something like `+-10` (an assumption by a user) or `+z1` (maybe a typo)